pyspelling/util/__init__.py

```diff
@@ -170,14 +170,22 @@ class Loader(loader):
 def read_config(file_name):
     """Read configuration."""
 
-    config = {}
     for name in (['.pyspelling.yml', '.spelling.yml'] if not file_name else [file_name]):
         if os.path.exists(name):
             if not file_name and name == '.spelling.yml':
                 warn_deprecated(
                     "Using '.spelling.yml' as the default is deprecated. Default config is now '.pyspelling.yml'"
                 )
             with codecs.open(name, 'r', encoding='utf-8') as f:
-                config = yaml_load(f.read())
-            break
-    return config
+                return yaml_load(f.read())
+    if not file_name:
+        file_name = '.pyspelling.yml'
+    print(
+        'Unable to load pyspelling configuration from %s, for more'
```

Comment:
I would prefer if this raised an exception instead. You can use the same text for the exception though.